docs_samples/Functions/tmp/data/a20f2b98-bd5a-4692-ab93-397a460d722f.txt

```diff
@@ -0,0 +1 @@
+Hello, World!
```

Comment:
What is this file for?

Reply:
Temp file to keep the \tmp\data folder structure intact\created for the repo.